### ISSUE TITLE
error dialog if the path to Lamdera binary is not set

### ISSUE THREAD

**Issue by frankbe** [intellij-elm/intellij-elm#809]
In the new beta version 5.0.0-beta26, when I open an existing elm project in Intellij for the first time, I get the prompt "No Elm projects found" with the option "Attach elm.json" as expected. After selecting the elm.json an error dialog appears (see picture), since I do not have Lamdera installed. 

![grafik](https://github.com/intellij-elm/intellij-elm/assets/914838/38c4e9b9-315a-4737-9ee7-c1404bc6fc3c)

PS: As a workaround, it was possible to set the lamdera binary path field in the settings to "/usr/bin/true".

**Comment by cies** [comment]
I used another workaround: set the path to `/usr/local/bin/elm` (in my case).

Would you like to take a stab at fixing it?

**Comment by frankbe** [comment]
yes, it's the same thing. I reviewed #791 and agree that the pull request most likely will fix this bug.
